tests/phpunit/CRM/Extension/MapperTest.php

```diff
@@ -5,6 +5,22 @@
  * @group headless
  */
 class CRM_Extension_MapperTest extends CiviUnitTestCase {
+
+  /**
+   * @var string
+   */
+  protected $basedir, $basedir2;
```

Comment:
I know it's testing code, but I haven't seen this syntax often in other parts of the code.